readinglist/__init__.py

```diff
@@ -39,4 +41,5 @@ def main(global_config, **settings):
         config.registry.settings['cliquet.paginate_by'] = 100
 
     config.scan("readinglist.views")
-    return config.make_wsgi_app()
+    app = config.make_wsgi_app()
```

Comment:
if that's merged in master - can we have the profiler activated only through a config option ?